### PR TITLE
UI changes, title

### PR DESCRIPTION
**Pull request by tenisakb**

### CODE REVIEW

File: src/components/landing-page/landing-page-slide.tsx [hunk 1]
```diff
@@ -334,295 +339,259 @@ const InfoHeaderContent = styled(Paragraph)`
...
-            </Content>
-          </ContentWrapper>
+const LandingPageSlide = ({ history, isLoggedIn }) => (
+  <ThemeProvider theme={theme}>
+    <ContainerWrapper>
+      <Header type={HEADER_TYPES.TEAM_PAGE} isLoggedIn={isLoggedIn} />
+      <HeaderContainer>
+        <Title>Private Cloud Storage, Powered by Crypto</Title>
+        <TitleSecondary>
+          Only your crypto key has access to your files
```
Comment (funkydrummer) [inline src/components/landing-page/landing-page-slide.tsx]: Change text to 'Only your crypto private key has access to you files'

File: src/components/landing-page/landing-page-slide.tsx [hunk 2]
```diff
@@ -334,295 +339,259 @@ const InfoHeaderContent = styled(Paragraph)`
...
+        </TitleSecondary>
+        <ContentWrapper>
+          <HeaderContent>
+            Easy and intuitive crypto cloud storage puts you in full control of
```
Comment (funkydrummer) [inline src/components/landing-page/landing-page-slide.tsx]: Change text to "Familiar cloud storage interface built with blockchain technology keeps you in full control of your files and who can view them."

File: src/components/shared/header.tsx [hunk 3]
```diff
@@ -177,10 +178,15 @@ const Header = ({ type, isLoggedIn }: HeaderProps) => {
...
           <StyledLink to={"/sign-up"}>
-            <CommunityButtonSecondary>Sign up</CommunityButtonSecondary>
+            <CommunityButtonSecondary>Explore Plans</CommunityButtonSecondary>
           </StyledLink>
-          <StyledLink to={"/login"}>
-            <CommunityButton>Login</CommunityButton>
+          <StyledLink
+            to={""}
+            onClick={() =>
+              window.open("https://www.kucoin.com/trade/OPQ-BTC", "_blank")
```
Comment (funkydrummer) [inline src/components/shared/header.tsx]: Keep Login button in the header. Kucoin link should replace login button in the landing page slide.

File: src/components/landing-page/landing-page-slide.tsx [hunk 4]
```diff
@@ -101,8 +101,8 @@ const Item = styled.div`
 `;
 
 const Button = styled.button`
-  width: 120px;
-  height: 40px;
+  width: 200px;
```
Comment (funkydrummer) [inline src/components/landing-page/landing-page-slide.tsx]: Keep button size and put 2nd button on right side. Change Login to Kucoin link for 2nd button.

File: src/components/landing-page/landing-page-slide.tsx (hunk 1 above)
Comment (tenisakb) [reply src/components/landing-page/landing-page-slide.tsx]: done

File: src/components/landing-page/landing-page-slide.tsx (hunk 2 above)
Comment (tenisakb) [reply src/components/landing-page/landing-page-slide.tsx]: done

File: src/components/shared/header.tsx (hunk 3 above)
Comment (tenisakb) [reply src/components/shared/header.tsx]: done

File: src/components/landing-page/landing-page-slide.tsx (hunk 4 above)
Comment (tenisakb) [reply src/components/landing-page/landing-page-slide.tsx]: done

**Pull request merged**